app/mailers/app_mailer.rb

```diff
@@ -3,4 +3,9 @@ def send_welcome_email(user)
     @user = user
     mail to: user.email, from: "info@myflix.com", subject: "Welcome to Myflix!"
   end
+
+  def send_forgot_password(user)
+    @user = user
+    mail to: user.email, from: "info@myflix.com", subject: "Password Reset Link"
```

Comment:
You can refactor `from:` to `default from:` above like a `before_action` read Rails API docs to see how to implement it.